test/e2e/storage/csi_volumes.go

```diff
@@ -47,6 +47,7 @@ var csiTestSuites = []func() testsuites.TestSuite{
 	testsuites.InitSubPathTestSuite,
 	testsuites.InitProvisioningTestSuite,
 	testsuites.InitSnapshottableTestSuite,
+	testsuites.InitMultiAttachTestSuite,
```

Comment:
nit: let's not call it multi attach. That will confuse it with RWX drivers.  Maybe multi mount? multi volumes?